test/debit.unit.js

```diff
@@ -97,8 +97,9 @@ describe('Storage/models/Debit', function() {
         amount: null
       });
 
-      newDebit.save(function(err, debit) {
-        expect(debit.amount).to.equal(0);
+      newDebit.save(function(err) {
+        expect(err).to.be.instanceOf(Error);
```

Comment:
@braydonf when you say "...gives better error messages" do you mean in the test output in the event of failure?